core/files/manager.php

```diff
@@ -85,34 +87,28 @@ public function on_export_post_meta( $skip, $meta_key ) {
 	 * @access public
 	 */
 	public function clear_cache() {
-		$files_classes = [
-			'post' => 'Files\CSS\Post',
-			'global' => 'Files\CSS\Global_CSS',
-			'custom' => 'Responsive\Files\Frontend',
-		];
+		// Delete post meta.
+		global $wpdb;
 
-		// Delete files.
-		$path = Base::get_base_uploads_dir() . Base::DEFAULT_FILES_DIR . '*';
-
-		foreach ( glob( $path ) as $file_path ) {
-			$file_name = basename( $file_path );
-
-			preg_match( '/^[a-z]+/', $file_name, $file_prefix );
+		$wpdb->delete(
+			$wpdb->postmeta, [
+				'meta_key' => Post_CSS::META_KEY,
+			]
+		);
 
-			$file_prefix = $file_prefix[0];
+		$wpdb->delete(
+			$wpdb->options, [
+				'option_name' => Global_CSS::META_KEY,
+			]
+		);
 
-			$class = 'Elementor\Core\\' . $files_classes[ $file_prefix ];
+		delete_option( Frontend::META_KEY );
 
-			if ( 'post' === $file_prefix ) {
-				preg_match( '/[0-9]+/', $file_name, $post_id_match );
-
-				$file_name = $post_id_match[0];
-			}
-
-			/** @var Base $file */
-			$file = new $class( $file_name );
+		// Delete files.
+		$path = Base::get_base_uploads_dir() . Base::DEFAULT_FILES_DIR . '*';
```

Comment:
rename `DEFAULT_FILES_DIR` => `FILES_DIR` it's not a default and cannot be replaced